src/utils/self_test/etc/selftest_expected.cnf

```diff
@@ -0,0 +1,536 @@
+
+#Default threshold scaling factor as percentage.
+[threshold]
+all=15
+bw=45
+tp=45
+min=45
+max=45
+med25=45
+med50=45
+med75=45
+sd=40
+av=40
+
+0:0-*:*-av=20
+0:0-0:0-av=20
+0:0-1:0-av=20
+0:0-2:0-av=20
+
+1:0-*:*-av=22
+1:0-0:0-av=22
+1:0-1:0-av=22
+1:0-2:0-av=20
+
+#Base default values
+[default_values]
+min=1400
```

Comment:
The units are what they always were (I believe that they are mill-seconds).  As far computing a "reasonable" value, it will depend on many factors such as transport method,  ofi interface, etc.  I've been executing the timings multiple times and then taking an approximation for the expected value.  In the future, this will be a job coming up with all the expected results.